src/pykx/pandas_api/pandas_indexing.py

```diff
@@ -453,6 +479,32 @@ def rename(self, labels=None, index=None, columns=None, axis=0,
                 t = _rename_columns(t, columns)
 
         return t
+    
+    def add_suffix(self, suffix=None, axis=0):
+        t = self
+        if suffix:
+            if axis == 0:
+                t = _pre_suf_fix_columns(t, suffix, suf=True)
+            elif axis == 1:
+                t = _pre_suf_fix_index(t, suffix, suf=True)
```

Comment:
I don't think we should support the conversions of this type for indexes unless the columns in the index are symbol/string types. Adding a suffix to integer/guid/float etc would have a number of implications on the symbol count within the processes which would have a memory impact. Additionally it will update the type of the columns which is possibly fine but probably worth requiring a user to convert to the symbol types before doing the index in that case

Reply:
We've decided to remove this logic and just fail with a `nyi` error for this particular situation.